plugins/automation/app/controllers/discourse_automation/admin_automations_controller.rb

```diff
@@ -37,69 +37,34 @@ def show
     def create
       automation_params = params.require(:automation).permit(:script, :trigger)
```

Comment:
I’m working on a new feature that will allow contracts to handle nested attributes, so we won’t have to do that anymore:
https://github.com/discourse/discourse/pull/36348